### ISSUE TITLE
Accessibility bug in the header logo

### ISSUE THREAD

**Issue by FranciscoImanolSuarez** [GoogleChrome/chromium-dashboard#5154]
Thank you for letting us know that something needs to be fixed. This page is for recording bugs on Chrome Status (https://www.chromestatus.com). If you need to report a problem with Chrome or Chromium, please use crbug (http://crbug.com/new). Thank you.

**Describe the bug**
The Chrome logo in the header, being descriptive, should have the alt empty, so that the screen reader understands that it is a decorative image.

**To Reproduce**
1. Go to /
2. Run axe developer tool
3. Locate the error: “Images must have alternative text”.

**Expected behavior**
Adding an empty tag to the img tag would solve the problem.

**Additional context**
https://www.w3.org/TR/WCAG20-TECHS/H37.html
https://dequeuniversity.com/rules/axe/4.1/image-alt

FIX PR: https://github.com/GoogleChrome/chromium-dashboard/pull/5155

**Comment by jrobbins** [comment]
The alt attribute was added.
